packages/bpk-component-banner-alert/stories.js

```diff
@@ -128,4 +203,50 @@ storiesOf('bpk-component-banner-alert', module)
       type={ALERT_TYPES.SUCCESS}
       icon={CurrencyIcon}
     />
+  ))
+  .add('All expandable', () => (
```

Comment:
Added this purely for debugging and testing out the styles will be removed when implementation of state colours is complete